apps/frontend/src/app/events/new/page.tsx

```diff
@@ -1,10 +1,52 @@
 'use client';
 
+import { zodResolver } from '@hookform/resolvers/zod';
 import { useState } from 'react';
+import { useForm } from 'react-hook-form';
+import { z } from 'zod';
+
+import { axiosConfig } from '@/api/axiosConfig';
+import { Form, FormControl, FormField, FormItem, FormLabel, FormMessage } from '@/components/ui/form';
+import { Input } from '@/components/ui/input';
+
+const formSchema = z.object({
+  name: z.string({
+    required_error: 'Ez a mező kötelező',
+    invalid_type_error: 'String, tesó!',
+  }),
+  location: z.string(),
+  startDate: z.coerce.date(),
+  endDate: z.coerce.date(),
+  description: z.string(),
+  drinkActions: z
+    .array(
+      z.object({
+        id: z.number(),
+        name: z.string(),
+        price: z.number().optional(),
+      })
+    )
+    .optional(),
+});
+
+async function onSubmit(values: z.infer<typeof formSchema>) {
```

Comment:
we should call `event.preventDefault()` at the begining of this function, so the standard HTML form is not submitted (you can see the URL change when you click save). Not sure how to get access to the event object in this function tho